bot/kodiak/event_handlers.py

```diff
@@ -174,7 +178,40 @@ def get_branch_name(raw_ref: str) -> Optional[str]:
             )
 
 
+_redis = None
```

Comment:
should this be a ContextVar rather than a global?

Reply:
we share this amongst tasks. Is that invalid?